drivers/flash/spi_nor.c

```diff
@@ -1236,6 +1297,39 @@ static int spi_nor_process_sfdp(const struct device *dev)
 				break;
 			}
 		}
+		if (id == JESD216_SFDP_PARAM_ID_4B_ADDR_INSTR) {
+			if (IS_ENABLED(ANY_INST_USE_4B_ADDR_OPCODES) &&
+			    DEV_CFG(dev)->use_4b_addr_opcodes) {
+				/*
+				 * Check table 4 byte address instruction table to get supported
+				 * erase opcodes when running in 4 byte address mode
+				 */
+				union {
+					uint32_t dw[2];
+					struct {
+						uint32_t dummy;
```

Comment:
When we probe the 4-byte Address Instruction Parameter Header, I think we should also check to see if the 1S-1S-1S Read, and Page Program commands are supported, IE check that bits 6 and 0 are set in this DWORD

Reply:
check added